### PR TITLE
Tezos mockup missing version constraint

### PR DESCRIPTION
**Pull request by pirbo**

### CODE REVIEW

File: packages/tezos-stdlib-unix/tezos-stdlib-unix.9.0/opam [hunk 1]
```diff
@@ -13,7 +13,10 @@ depends: [
   "ptime" { >= "0.8.4" }
   "mtime" { >= "1.0.0" }
   "conf-libev"
+  "domain-name" { < "0.3.1" }
   "ipaddr" { >= "4.0.0" }
+  "ezjsonm" { >= "1.1.0" }
+  "fmt" { >= "0.8.7" }
```
Comment (kit-ty-kate) [inline packages/tezos-stdlib-unix/tezos-stdlib-unix.9.0/opam:19]: Could you move these in the `conflicts` section? As I understand it they’re not explicit dependencies and only used implicitly through other libraries via domain-name

Comment (pirbo) [reply packages/tezos-stdlib-unix/tezos-stdlib-unix.9.0/opam]: Done

File: packages/tezos-stdlib-unix/tezos-stdlib-unix.9.1/opam [hunk 2]
```diff
@@ -13,7 +13,10 @@ depends: [
   "ptime" { >= "0.8.4" }
   "mtime" { >= "1.0.0" }
   "conf-libev"
+  "domain-name" { < "0.3.1" }
   "ipaddr" { >= "4.0.0" }
+  "ezjsonm" { >= "1.1.0" }
+  "fmt" { >= "0.8.7" }
```
[the remark this answers is not in the document]
Comment (pirbo) [reply packages/tezos-stdlib-unix/tezos-stdlib-unix.9.1/opam]: indeed

File: packages/tezos-stdlib-unix/tezos-stdlib-unix.9.0/opam (hunk 1 above)
Comment (kit-ty-kate) [reply packages/tezos-stdlib-unix/tezos-stdlib-unix.9.0/opam]: Same for `domain-name`

Comment (pirbo) [reply packages/tezos-stdlib-unix/tezos-stdlib-unix.9.0/opam]: Right...

**Comment by samoht** [comment]
Thanks!

**Pull request merged**